Month1/Week2/Section2/Month1-Week2-Section2/Spotify/PlayListContainer.cpp

```diff
@@ -0,0 +1,25 @@
+#include"PlayListContainer.h"
+
+FPlayListContainer::FPlayListContainer()
+{
+	PlayListCount = 0;
+}
+
+FPlayListContainer::FPlayListContainer(FPlayList NewPlayList)
+{
+	PlayLists[PlayListCount] = NewPlayList;
+	PlayListCount++;
```

Comment:
Careful. PlaylistCount has not been initialized at this point, it could be any random value. PlaylistCount++ is not guaranteed to be 1 every time.